lib/media_types/minitest/test_factory.rb

```diff
@@ -0,0 +1,20 @@
+require 'minitest/autorun'
+
+class Minitest::Test < Minitest::Runnable
+  include MediaTypes::Assertions
+  def self.build_fixture_tests(type)
```

Comment:
```suggestion

  def self.build_fixture_tests(klazz)
```

The canonical name for a type isn't `type` but `klazz`.

Reply:
This method reads as pretty complex. Probably a way to simplify it or extract commonalities